src/tir/schedule/analysis/analysis.cc

```diff
@@ -45,6 +45,26 @@ const PrimFuncNode* GetRootPrimFunc(const IRModule& mod, const StmtNode* root_bl
   throw;
 }
 
+const PrimFuncNode* GetPrimFuncFromSparseBlock(const IRModule& mod, const SparseBlockNode* sp_block,
```

Comment:
What's this function used for?

Reply:
It's used for finding the PrimFunc that a given SparseBlock is in.

During scheduling we don't actually know in which PrimFunc we are manipulating. Therefore, we may need such a helper to fetch the PrimFunc and its corresponding GlobalVar in the IRModule.